### DISCUSSION TITLE
pull request closed token not validated? why & help

### DISCUSSION THREAD

**Discussion by shhitzelectric** [jup-ag/token-list#1247] [Q&A]
I recently opened a pull request to validate a token and it had been on the list for a couple of weeks and it was just recently closed I do not know why and I would like some help trying to fix this can somebody please help. @9yoi

**Comment by Tahititoken** [comment]
hellllllllllp

**Comment by ghost** [comment]
Be sure that you complied with all requirements 
https://station.jup.ag/docs/get-your-token-onto-jup

**Comment by lancoer** [comment]
@shhitzelectric Have you checked the discord?
You should alert the team to list your token.
